jinja/carnival/prize_booth.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'base.html' %}
+{% block content %}
+{% block navbar %}
+{% include "navbar.html" %}
+{% endblock %}
+
+<div class="parallax" style="background-image: url('../img/xanadu-background.png'); box-shadow: none;">
+    <div class="container">
+
+        <div class="mt-5 pt-5 text-center teal-text text-lighten-2">
+            <img src="../img/carnival_logo_splash.png" class="logo"></img>
+        </div>
+        <div class="row-sm text-center justify-content-center" style="color:grey; font-family: 'Roboto', sans-serif !important;">
+            <h3 class="col-12 mt-5 mb-5 grey-text">
+                Exchange your tickets for amazing <strong>Carnival Prizes</strong>. 
+            </h3>
+            
+	<div style="font-size: 14pt;">
+            <p>
+            The participants with the most points will be the winners of some cool Xanadu swag. 😎
+
+</p>
+
+<p>
+Make sure to <a href="https://pennylane.ai/carnival/register.html">register</a> and save all of your ticket codes. 
+At the end of the month you will receive a form to submit your codes and redeem PennyLane #QuantumCarnival prizes! 🎟️
+
+</p>
+
+<p>
+What kind of things can I get? Check-out all of the cool items below. 🤩

```

Comment:
Just above we're using the second person (you/your), but here is the first person.

Reply:
```suggestion
What kind of things can I get? Check out all of the cool items below. 🤩
```